tests/security/pam/pam_config.pm

```diff
@@ -0,0 +1,57 @@
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: PAM tests for pam-config, create, add or delete service
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#70345, tc#1167579
```

Comment:
the tc#1167579 number seems incorrect.

Reply:
done

Reply:
@rfan1 I just see the latest code is still tc#1167579, or maybe it is not applied successfully? 
It should be tc#1767580, am I right ?

Reply:
oops, let me fix it, I remembered that I have modified it

Reply:
it is 1767580 already, please double check

Reply:
I just check this , https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10989/files#diff-0e4ffde6b4f071e6b09a0bb176b6eb0c

Reply:
done :)